sympy/stats/crv_types.py

```diff
@@ -2940,6 +2942,92 @@ def Pareto(name, xm, alpha):
 
     return rv(name, ParetoDistribution, (xm, alpha))
 
+#-------------------------------------------------------------------------------
+# PowerFunction distribution ---------------------------------------------------
+
+
+class PowerFunctionDistribution(SingleContinuousDistribution):
+    _argnames=('alpha','a','b')
+
+    @property
+    def set(self):
+        return Interval(self.a, self.b)
+
+    @staticmethod
+    def check(alpha, a, b):
+        _value_check(a.is_real, "Continuous Boundary parameter should be real.")
+        _value_check(b.is_real, "Continuous Boundary parameter should be real.")
+        _value_check(a < b, " 'a' the left Boundary must be smaller than 'b' the right Boundary." )
+        _value_check(alpha.is_positive, "Continuous Shape parameter should be positive.")
+
+    def pdf(self, x):
+        alpha, a, b = self.alpha, self.a, self.b
+        num = alpha*(x - a)**(alpha - 1)
+        den = (b - a)**alpha
+        return num/den
+
+def PowerFunction(name, alpha, a, b):
+    r"""
+    Creates a continuous random variable with a Power Function Distribution
+
+    .. math::
+        f(x) := \begin{cases}
+                 \frac{{\alpha}(x - a)^{\alpha - 1}}{(b - a)^{\alpha}} & \text{for } x \in [a,b]  \\
+                  0               & \text{otherwise}
+                \end{cases}
```

Comment:
Probably, we can replace this with only the pdf rather than showing 0 in case x is not in [a, b] as support is already defined below.

Reply:
Once this is done, we can merge this.